site/components/ReviewConsequenceTable.tsx

```diff
@@ -45,10 +45,7 @@ export const createChangeLink = (
                 pathname: `${href}/${disruptionId}${index !== undefined ? `/${index}` : ""}`,
                 query: includePreviousPage
                     ? {
-                          return:
-                              isDisruptionDetail || isTemplate
-                                  ? DISRUPTION_DETAIL_PAGE_PATH
-                                  : REVIEW_DISRUPTION_PAGE_PATH,
+                          return: isDisruptionDetail ? DISRUPTION_DETAIL_PAGE_PATH : REVIEW_DISRUPTION_PAGE_PATH,
```

Comment:
If you create a new template and navigate to review template page and click cancel button - we are redirected to disruption details page while it should be redirected to view all templates page

Reply:
When I follow this path I get redirected to view all templates not disruption details - can you demo this to me please?